tests/functional/fixtures/ext/foo/bar/ucp/main_info.php

```diff
@@ -19,9 +19,9 @@ function module()
 	{
 		return array(
```

Comment:
Would also opt for changing this file to short array syntax while you're changing it